numpy/core/_internal.py

```diff
@@ -355,15 +355,37 @@ def _view_is_safe(oldtype, newtype):
         If the new type is incompatible with the old type.
 
     """
+    from numpy.lib.type_check import find_dtype_offsets
```

Comment:
This import feels kinda hacky, but it can't go globally. Should I put this function somewhere else?